### ISSUE TITLE
Implement defaultLiteral for InputList

### ISSUE THREAD

**Issue by jgreben** [LD4P/sinopia_editor#265]
e.g.
```
"defaults": [
          {
            "defaultURI": "http://id.loc.gov/vocabulary/carriers/nc",
            "defaultLiteral": "volume"
          }
```

**Comment by jgreben** [comment]
See (e.g.) https://github.com/LD4P/sinopia_editor/pull/246/commits/31630430c3180c838267511d164b22029a9e5792

**Comment by jgreben** [comment]
Fixed by #274
